Bindings/Java/Matlab/Utilities/osimList.m

```diff
@@ -0,0 +1,95 @@
+classdef osimList
+properties
+        model % a reference to an opensim model
+        list; % an opensim list
+    end
+    
+    methods 
+        function obj = osimList(model,classname)
```

Comment:
We could add support for arbitrary component types using your `eval()` and `getConcreteClassName()` trick. It won't always work though.